tests/src/trivia.rs

```diff
@@ -0,0 +1,16 @@
+#![allow(dead_code)]
+/// The eth_abi marco generates code uses
```

Comment:
The comment is too specific, it shouldn't be doc comment also

I would describe it as "minimal showcase of code generator for ethereum abi"